html/includes/authentication/active_directory.inc.php

```diff
@@ -4,10 +4,14 @@
 
 // disable certificate checking before connect if required
 if (isset($config['auth_ad_check_certificates']) &&
-          $config['auth_ad_check_certificates'] == 0) {
+          !$config['auth_ad_check_certificates']) {
```

Comment:
Users have `$config['auth_ad_check_certificates'] = 1` in their configs right now.  I wanted to support true and false too.